core/src/main/proto/xtdb/meta/block/proto/block.proto

```diff
@@ -2,11 +2,14 @@ edition = "2023";
 
 package xtdb.block.proto;
 
+import "xtdb/log/proto/log.proto";
+
 option java_multiple_files = true;
 
 message TableBlock {
   bytes arrow_schema = 1;
   int64 row_count = 2;
+  repeated xtdb.log.proto.AddedTrie current_tries = 3;
```

Comment:
Not sure we want to reuse `AddedTrie` here, but it was close enough for my purposes so that I reused it.

Reply:
yeah, seems reasonable. maybe a rename then? `TrieDetails`? :thinking:

Reply:
Table Name is kind of redundant in that message.